src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $daysGap;
+  width: ($cellWidth * 7) + ($calendarPadding * 2) + ($daysGap * 6);
```

Comment:
The calculation of the calendar width is a bit complex. It would be better to calculate the width of a single cell with gaps and then multiply it by 7. This would make the code more readable and maintainable.